apis/python/tests/test_context.py

```diff
@@ -131,3 +132,16 @@ def test_replace_config_after_construction():
             new_tdb_ctx = new_soma_ctx.tiledb_ctx
         mock_ctx.assert_called_once()
         assert new_tdb_ctx.config()["vfs.s3.region"] == "us-west-2"
+
+
+def test_malformed_concurrency_config_value():
+    import numpy as np
```

Comment:
Why import in the test instead of up top?

Reply:
Just trying to isolate test dependencies.  But not a major issue either way....